db/migrate/20220719151539_add_physical_storage_to_event_streams.rb

```diff
@@ -0,0 +1,5 @@
+class AddPhysicalStorageToEventStreams < ActiveRecord::Migration[6.0]
+  def change
+    add_column :event_streams, :physical_storage_id, :bigint
+  end
```

Comment:
@TonySchneider `process_object_in_event` also sets the `name` so best to add `add_column :event_streams, :physical_storage_name, :string` here as well